Automattic/woocommerce-payments
# feat: add advanced settings section

Fixes https://github.com/Automattic/woocommerce-payments/issues/1688

#### Changes proposed in this Pull Request









Adds the "Advanced settings" section to the settings page.

#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Go to your WCPay Home: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- Section is displayed

![2021-05-13 16 21 03](https://user-images.githubusercontent.com/273592/118189341-41676d00-b407-11eb-8cfa-4b8437ed8626.gif)


-------------------
